doc/manuals/admin/cli.md

```diff
@@ -138,3 +138,5 @@ The list of available options is the following:
 -   **-reqPoolSize**. Size of thread pool for incoming connections. Default value is 0, meaning *no thread pool*.
 -   **-statCounters**, **-statSemWait**, **-statTiming** and **-statNotifQueue**. Enable statistics
     generation. See [statistics documentation](statistics.md).
+-   **-notifErrLogInterval**. Interval to issue warning log about notification errors - in number of errors.
```

Comment:
Do we really want to open this to external user configuration? I have though in these parameters as configurable aspects in the alarm manager but "hardwired" to an specific value (0?) in the initializaiton of such manager.